server/uno.go

```diff
@@ -38,14 +38,25 @@ import (
 // var allCards map[string][]model.Card = make(map[string][]model.Card) // k: username, v: list of cards
 // var gameStarted bool = false
 
-////////////////////////////////////////////////////////////
-// Utility functions
-////////////////////////////////////////////////////////////
 // func newRandomCard() []model.Card {
 // TODO use deck utils instead
 // 	return []model.Card{model.Card{rand.Intn(10), randColor(rand.Intn(4))}}
 // }
 
+////////////////////////////////////////////////////////////
+// Utility functions
```

Comment:
I could be wrong... But I wonder if it makes sense long term to start breaking utility methods into a separate file to start creating boundaries between interactions?

I could also see breaking the files down into `card.go` `player.go` and game.go`.  And leaving `uno.go` to orchestrate for them.

However, these are just my thoughts. I just wonder if it would start to `spread the table space` as Jacob said in class.

Reply:
That is usually how i end up organizing items. I do like this idea.